src/main/java/com/commercetools/sync/commons/utils/AssetsUpdateActionUtils.java

```diff
@@ -21,14 +25,21 @@
 
 import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
 import static com.commercetools.sync.commons.utils.OptionalUtils.filterEmptyOptionals;
+import static java.lang.String.format;
 import static java.util.Collections.singletonList;
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.toCollection;
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
 
 public final class AssetsUpdateActionUtils {
 
+    private static final String ASSET_KEY_NOT_SET = "Asset with %s has no key set. Keys are required for "
```

Comment:
```suggestion
    private static final String ASSET_KEY_NOT_SET = "Asset with %s has no defined key. Keys are required for "
```

Reply:
I guess, it's better to use the same convention/similar messages with the other errors, so I would go with Andi's message.
Another example for Category key, `Category with id: '%s' has no key set. Keys are required for category matching`